common/utils/licenses.js

```diff
@@ -69,47 +67,84 @@ export const licenseMap = {
     ],
   },
   'cc-by-nd': {
-    url: 'https://creativecommons.org/licenses/by-nd/4.0/',
-    text: 'CC BY-ND',
     icons: ['cc', 'ccBy', 'ccNd'],
     description: 'Free to use with attribution. No modifications permitted.',
     humanReadableText: [],
   },
   'cc-by-sa': {
-    url: 'https://creativecommons.org/licenses/by-sa/4.0/',
-    text: 'CC BY-SA',
     icons: ['cc', 'ccBy'],
+    description: null,
     humanReadableText: [],
   },
   'cc-by-nc-sa': {
-    url: 'https://creativecommons.org/licenses/by-nc-sa/4.0/',
-    text: 'CC BY-NC-SA',
     icons: ['cc', 'ccBy', 'ccNc'],
+    description: null,
     humanReadableText: [],
   },
   ogl: {
-    url:
-      'http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/',
-    text: 'Open Government License',
     icons: [],
+    description: null,
     humanReadableText: [],
   },
   opl: {
-    url: 'http://opencontent.org/openpub/',
-    text: 'Open Publication License',
     icons: [],
+    description: null,
     humanReadableText: [],
   },
   inc: {
-    url: 'http://rightsstatements.org/vocab/InC/1.0/',
-    text: 'In copyright',
     icons: [],
+    description: null,
     humanReadableText: [],
   },
 };
 
-export default function getLicenseInfo(
-  licenseIdentifier: string
-): ?LicenseData {
-  return licenseMap[licenseIdentifier.toLowerCase()];
+export default function getAugmentedLicenseInfo(
+  license: LicenseAPIData
+): LicenseUIData {
+  const additionalLicenseData = UILicenseMap[license.id.toLowerCase()];
+  return {
+    ...license,
+    ...additionalLicenseData,
+  };
+}
+
+// Content editors are restricted to the following licenses
```

Comment:
Why not update tzizit with this?
https://github.com/wellcomecollection/tzitzit

Reply:
Could do that, just thought it might be adding noise for the content editors, giving them a load of license options they've never had a need for.